src/sessions/builder.rs

```diff
@@ -2,6 +2,8 @@ use std::collections::HashMap;
 
 use instant::Duration;
 
+#[allow(unused_imports, reason = "used only in doc comment")]
```

Comment:
Not a huge deal but to avoid the warning suppression you can do this in doc comments:
```rust
/// ... detect events such as [`MismatchedChecksum`](crate::GgrsEvent::MismatchedChecksum)
```